packages/block-library/src/columns/variations.js

```diff
@@ -12,6 +12,28 @@ import { __ } from '@wordpress/i18n';
  * @type {WPBlockVariation[]}
  */
 const variations = [
+	{
+		name: 'one-column-full',
+		title: __( '100' ),
+		description: __( 'One column; full width' ),
```

Comment:
This is open to interpretation, but 'full width' could end up being confused for the alignment option (especially for screenreader users). I think just leaving it at 'One column' would be fine, but there might also be phrasing like 'full size' or '100%' that could work.

Reply:
```suggestion
		description: __( 'One column' ),
```